systemtests/src/test/java/io/enmasse/systemtest/common/catalog/ServiceCatalogWebTest.java

```diff
@@ -74,7 +74,6 @@ void testProvisionAddressSpaceBrokered() throws Exception {
         String namespace = getUserProjectName(brokered);
         provisionedServices.put(namespace, brokered);
         OpenshiftWebPage ocPage = new OpenshiftWebPage(selenium, addressApiClient, getOCConsoleRoute(), developer);
-        ocPage.openOpenshiftPage();
```

Comment:
seems to me like this row shouldn't be removed

Reply:
:/ youre right man, my mistake, added again.